### PR TITLE
basic stylesheet

### PR DESCRIPTION
**Pull request by bobbybwoy**
## Things to Check
- [ ] Have all the requirments of your ticket have been met - this is a work-in-progress
- [ ] Have you added tests if needed for your ticket - no tests have been added
- [ ] Are all the tests passing - N/A

### CODE REVIEW

**Review by Kiran-R-K — COMMENTED** [review]
This is looking good - I've left a question about naming (feel free to let me know if you disagree with my comment - this can be a conversation)
you marked this as an in progress review in your description so let me know when it's ready for a final review

File: src/App.scss [hunk 1]
```diff
@@ -1,38 +1,42 @@
...
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.secondary {
```
Comment (Kiran-R-K) [inline src/App.scss]: it's not clear to me what type of class secondary is referring to here - perhaps it could have a more descriptive name?

Comment (bobbybwoy) [reply src/App.scss]: Changed the name of the property to be more explicit. I misunderstood that this was the root stylesheet, and that everyone else component will have their own stylesheets. So, this is no longer a work-in-progress.

**Review by Kiran-R-K — APPROVED** [review]
looks good - ready to merge

**Pull request merged**